src/main/resources/locales/en/translation/gender-details.json

```diff
@@ -21,6 +21,12 @@
   },
   "otherTitlePreference": "If 'other' please enter your preferred title",
   "errors": {
+    "gender": {
+      "required": "Select an option for the \"What is you sex\" question"
+    },
+    "preferredTitle": {
+      "required": "Select a list option from the \"What is your preferred title\""
```

Comment:
The word "the" is extra here? Either "Select a list option from the 'What is your preferred title?' dropdown" or just "Select a list option from 'What is your preferred title?'